### PR TITLE
powershell: initial package release

### LINKED ISSUE #4076: [Package Request] PowerShell
Name: PowerShell
Description: Microsoft PowerShell
Website: https://microsoft.com/powershell
Software documentation: https://docs.microsoft.com/powershell
Build/Installation documentation:  https://github.com/PowerShell/PowerShell
Source Code: https://github.com/PowerShell/PowerShell
License:  MIT license.

### PR DESCRIPTION
**Pull request by hgy59**
_Motivation:_  PowerShell is the first package depending on .NET 6 runtime
_Linked issues:_  #4976, closes #4076

### Checklist
- [ ] Build rule `all-supported` completed successfully
- [ ] Package upgrade completed successfully
- [ ] New installation of package completed successfully

### CODE REVIEW

File: cross/dotnet-runtime/Makefile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+PKG_NAME = aspnetcore-runtime
```
Comment (publicarray) [inline cross/dotnet-runtime/Makefile:1]: Can we drop the `asp` and `core`? AFAIK the package is just called `.net` or `dotnet` with just 2 variations either SDK or runtime

I thought the `asp` part is the proprietary windows APIs

Comment (hgy59) [reply cross/dotnet-runtime/Makefile:1]: not it isn't  (see https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
There are three variants of runtimes to download
-  `ASP.NET Core Runtime` including `.NET Runtime`
-  `.NET Desktop Runtime`  including `.NET Runtime` (Windows only)
-  `.NET Runtime` only

**Comment by hgy59** [comment]
Output of `dotnet --info`
```
$ dotnet --info

Host (useful for support):
  Version: 6.0.1
  Commit:  3a25a7f1cc

.NET SDKs installed:
  No SDKs were found.

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.1 [/volume1/@appstore/dotnet-runtime/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.1 [/volume1/@appstore/dotnet-runtime/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download

```

File: cross/dotnet-runtime/Makefile (hunk 1 above)
Comment (hgy59) [reply cross/dotnet-runtime/Makefile:1]: @publicarray  sorry, just cherry picked a commit of #4976 to fix the build.
I will now remove dotnet runtime from the PR...

**Comment by hgy59** [comment]
closing due inactivity...

**Pull request closed without merging**